svgnative/src/SVGDocument.cpp

```diff
@@ -126,6 +126,22 @@ void SVGDocument::Render(const char* id, const ColorMap& colorMap, float width,
     mDocument->Render(id, colorMap, width, height);
 }
 
+Rect SVGDocument::Bounds()
+{
+    // TODO: Should we return an exception or fire an assertion for this or just return an empty rectangle?
+    if (!mDocument)
+        return Rect{};
```

Comment:
@dirkschulze What do you suggest about this? In such cases, should we return an empty rectangle or throw an exception? The thing is that giving an empty rectangle is kind of misleading.

Reply:
Yeah, I know what you mean. Alternatively, you could pass a Rect parameter by-reference and return a boolean to indicate success.

But IMO an empty rectangle also indicates that there is an issue.